workshop_attendance.txt

```diff
@@ -1,4 +0,0 @@
-# Please enter your name and your github handle, each in a new line.
-
-Blanca Mancilla
-@bluciam
```

Comment:
Oh no! You deleted the original file :-( Is it possible to revert the deletion of `workshop_attendance.txt`?